gradient_boosting.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+
+from sklearn.base import RegressorMixin
+from tree import DecisionTreeRegressor
+
+
+class BaseGradientBoostingRegressor:
+
+    def __init__(self,
+                 n_estimators: 5,
+                 max_depth: int,
+                 learning_rate=None):
```

Comment:
Does it work if you do not specify `learning_rate` parameter? I mean, that if `learning_rate` is `None`, will the code be runnable?

Reply:
No, it will not run